support/pg_versions

```diff
@@ -1 +1 @@
-12.1
+9.6.17;10.12;11.7;12.2
```

Comment:
I don't really understand this change. Why is not:

```diff
- 9.6.12;10.7;11.2;12.1
+ 9.6.17;10.12;11.7;12.2
```

Reply:
This `pg_versions` file was introduced in #32 where a small refactoring of the build script was done, and pg 12.1 was built to set it as the new default version.

Reply:
Ah ok, sorry, I missed the previous PR. So this file is just used at build time and contains the pg versions we last built. In #32 we only needed 12.1 but now we need all the rest. I see.

Reply:
> Ah ok, sorry, I missed the previous PR. 

NP, I should have explicitly linked to #32 for clarity.

> So this file is just used at build time and contains the pg versions we last built. In #32 we only needed 12.1 but now we need all the rest. I see.

Exactly that.